src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -197,6 +204,25 @@ public Resource getFlatFileByArtefactID(UUID artefactId, Boolean verification) {
         return azureBlobService.getBlobFile(sourceArtefactId, provenance);
     }
 
+    /**
+     * Checks if the artefact has a display from date of today or previous then triggers the sub fulfilment
+     * process on subscription-management if appropriate.
+     */
+    public void checkAndTriggerSubscriptionManagement(Artefact artefact) {
+        //TODO: fully switch this logic to localdates once artefact model changes
+        if (artefact.getDisplayFrom().toLocalDate().isBefore(LocalDate.now().plusDays(1))
+            && (artefact.getDisplayTo() == null
+            || artefact.getDisplayTo().toLocalDate().isAfter(LocalDate.now().minusDays(1)))) {
+            log.info(sendArtefactForSubscription(artefact));
+        } else {
+            log.error("invalid publication, no trigger sent");
```

Comment:
I'm not sure we want to log an error here, as a publication in the future wouldn't be an error scenario (just picked up later)

Reply:
Have now switched it to an info log - is that okay?

Reply:
do we even need to log this non action? 🤔

Reply:
Can just chuck it then! No worries :)